pom.xml

```diff
@@ -453,6 +455,11 @@
             <artifactId>jackson-dataformat-yaml</artifactId>
             <version>${lib.jackson.version}</version>
         </dependency>
+        <dependency>
+            <groupId>io.minio</groupId>
+            <artifactId>minio</artifactId>
+            <version>8.5.7</version>
```

Comment:
Nit: Please use a `lib.minio.version` property here to be consistent with how it's done for other dependencies.